Chirp/test/RepositoryTests/AuthorRepositoryUnitTest.cs

```diff
@@ -1,6 +1,138 @@
-﻿namespace RepositoryTests;
+﻿using Chirp.Razor.DataModels;
+using Microsoft.Data.Sqlite;
+using Microsoft.EntityFrameworkCore;
+using Moq;
+using SimpleDB;
+using SimpleDB.Model;
 
-public class AuthorRepositoryUnitTest
+namespace RepositoryTests;
+
+public class AuthorRepositoryUnitTest : IDisposable
 {
+    private readonly SqliteConnection _connection;
+
+    public AuthorRepositoryUnitTest()
+    {
+        _connection = new SqliteConnection("Filename=:memory:");
+        _connection.Open();
+    }
+
+    private ChirpDBContext GetContext()
+    {
+        var builder = new DbContextOptionsBuilder<ChirpDBContext>().UseSqlite(_connection);
+
+        var context = new ChirpDBContext(builder.Options);
+        context.Database.EnsureDeleted(); //Ensures that any test runs on their own DB
+        context.Database.EnsureCreated(); // Applies the schema to the database
+
+        return context;
+    }
+
+    [Fact]
+    public void GetAuthorByName_NameCantBeFound_ReturnNull()
+    {
```

Comment:
This should call getAutherByName with another value than the one that is inserted, and also await the result